collector/docker/kindling-collector-config.yml

```diff
@@ -42,8 +42,10 @@ analyzers:
   tcpmetricanalyzer:
   networkanalyzer:
     connect_timeout: 100
+    # How many seconds to wait until we consider a request as complete.
+    request_timeout: 15
```

Comment:
I think "request_timeout" is ambiguous now because it is used to check whether `fds` is reused. Better change to `fd_reuse_timeout`.